applications/portfolioDrafts/portfolio/createPortfolioStep.ts

```diff
@@ -1,12 +1,14 @@
-import { UpdateCommand } from "@aws-sdk/lib-dynamodb";
+// import { UpdateCommand } from "@aws-sdk/lib-dynamodb";
+// import { portfolioStepCommand } from "../utils/commands";
+import { createPortfolioStepCommand } from "../utils/commands/createPortfolioStepCommand";
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
 import { ErrorCodes } from "../models/Error";
 import { PortfolioStep } from "../models/PortfolioStep";
 import { dynamodbClient as client } from "../utils/dynamodb";
 import { ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode } from "../utils/response";
-import { isPortfolioStep, isValidJson } from "../utils/validation";
+import { isPortfolioStep, isValidJson, isBodyPresent, isPathParameterPresent } from "../utils/validation";
 
-const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+const TABLE_NAME = process.env.ATAT_TABLE_NAME ?? "";
```

Comment:
If this isn't set, we need to totally abort. We can't work with a default value. This would be an unexpected infrastructure issue. We probably want a helper that does something like:

```
if (!process.env.ATAT_TABLE_NAME) {
  throw new Error("Oh no! The table name isn't set.")
}
```

of course with a better message and maybe a subclass of `Error`? But this really is an exceptional case that should never happen and if it does, we need to return a 500 response with a really good log message ASAP.